### ISSUE TITLE
Conda-forge build does not include eingen source folder

### ISSUE THREAD

**Issue by pfmephisto** [wouterboomsma/eigency#66]
The current build on conda-forge is still version 3.4.0.0 and does not include the eigen source folder.

**Comment by BTOdell** [comment]
We don't support or maintain the conda-forge build. I believe @fbordeu is currently maintaining that distribution using his own fork of Eigency. I'm sure he'll be updating the conda-forge version soon since he was the one to submit the memory leak patch that made it into v3.4.0.1.

Also, the conda-forge version doesn't include an embedded eigen library because it uses the system install from conda-forge, as far as I'm aware.

**Comment by fbordeu** [comment]
you are correct @BTOdell, I maintain the package but from this repo (I use the fork only for pull request and for debuging).
I just update the package, took me all day to test my library for other memory leak (mainly sympy is leaking). If you need to build packaged using only conda packages, I recommend installing the conda package "eigen".

![image](https://github.com/wouterboomsma/eigency/assets/5357208/32bd825e-ec76-46b9-a65a-881e0e8c958e)
 
@BTOdell  I need to pull some changes to eigency to make the repository 100% conda build compatible (for the moment patch the sources before building the package). 

More info on :
https://github.com/conda-forge/eigency-feedstock/tree/main/recipe

**Comment by fbordeu** [comment]
closing this issue, @pfmephisto you can reopen this or another issue if your problem is not solverd
